esp32/modules/tasks/powermanagement.py

```diff
@@ -16,9 +16,9 @@ def pm_task():
     global requestedStandbyTime
     
     idleTime = virtualtimers.idle_time()
-    print("[Power management] Next task wants to run in "+str(idleTime)+" ms.")
+    #print("[Power management] Next task wants to run in "+str(idleTime)+" ms.")
         
-    if idleTime>30000:
+    if idleTime > 30000 and ( badge.usb_volt_sense() < 4500 or badge.nvs_get_u8('badge', 'usb_stay_awake', 0) == 0 ):
```

Comment:
I think so. It's how it's done for the "USB connected" image on the display (in `easydraw.battery()`). It could be nice to refactor that check into a badge.usb_attached() or something.